realm/realm-library/src/objectServer/java/io/realm/internal/ResultHandler.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.internal;
```

Comment:
I would suggest moving this to `io.realm.internal.network`. It would make it more clear what the class is used for. The name `ResultHandler` is very generic when living directly in `io.realm.internal`